src/CommandLine.Tests/packages.config

```diff
@@ -7,4 +7,5 @@
   <package id="xunit.core" version="2.0.0" targetFramework="net45" />
   <package id="xunit.extensibility.core" version="2.0.0" targetFramework="net45" />
   <package id="xunit.extensions" version="2.0.0" targetFramework="net45" />
+  <package id="xunit.runner.visualstudio" version="2.0.0" targetFramework="net45" />

```

Comment:
I just pushed this in 4a155be, so just rebase this branch on top of master and that line should go away (since it is already added). I think we prefer rebases over merges to keep a linear history but I'm not sure so we might want a comment from @gsscoder here.